sprint-poker-springboot/src/main/java/org/zhxie/sprinpoker/controller/ResponseResult.java

```diff
@@ -0,0 +1,50 @@
+package org.zhxie.sprinpoker.controller;
+
+import com.google.common.collect.Maps;
+
+import java.util.Map;
+
+/**
+ * Created by jianyang on 2019/1/9.
+ */
+public class ResponseResult {
+    private boolean isSuccess;
```

Comment:
use a int to indicate the flag instead of a boolean

Reply:
ok, i will change